trellis/core/logging.cpp

```diff
@@ -0,0 +1,47 @@
+/*
+ * Copyright (C) 2021 Agtonomy
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+#include "logging.hpp"
+
+#include <ecal/ecal.h>
+
+namespace trellis {
+namespace core {
+namespace Log {
+
+namespace {
+
+eCAL_Logging_eLogLevel LogLevelToEcalLevel(LogLevel level) {
+  return std::unordered_map<LogLevel, eCAL_Logging_eLogLevel>{
+      {kInfo, log_level_info},   {kWarn, log_level_warning}, {kError, log_level_error},
+      {kFatal, log_level_fatal}, {kDebug, log_level_debug1},
```

Comment:
Might be better to statically define this map so it doesn't have to be instantiated every time?